host/frontend/host-orchestrator/orchestrator/instancemanager_test.go

```diff
@@ -291,8 +293,7 @@ func TestLaunchCVDProcedureBuilder(t *testing.T) {
 }
 
 func TestStageDownloadCVDDownloadFails(t *testing.T) {
-	dir := t.TempDir()
-	cvdBin := dir + "/cvd"
+	cvdBin := "/tmp/cvd"
```

Comment:
use `os.TempDir()` instead of hardcoded `"/tmp"`.

Reply:
Done